scripts/install_libseccomp.sh

```diff
@@ -25,13 +25,14 @@ function build_and_install_gperf() {
 
     echo "Build and install gperf version ${gperf_version}"
     gperf_install_dir="$(mktemp -d --tmpdir build-gperf.XXXXX)"
+	chmod o+rx $gperf_install_dir
     curl -sLO "${gperf_tarball_url}"
     tar -xf "${gperf_tarball}"
     pushd "gperf-${gperf_version}"
     ./configure --prefix="${gperf_install_dir}"
     make
     make install
-    export PATH=$PATH:"${gperf_install_dir}"/bin
+	export PATH=$PATH:"${gperf_install_dir}"/bin
```

Comment:
Thanks for catching these!

Reply:
Wait actually isn't this still needed? At least one of the CI steps should not install gperf via apt so we can test this path, so I'll make that change.

Reply:
https://github.com/libseccomp-rs/libseccomp-rs/runs/5382938044

yes, this line is still required when using the script to build and install gperf. In order to test this functionality, I'll remove the apt install gperf from the code coverage step since there's no matrix (1 run instead of 3) and the code coverage step takes longer anyway, so the increase in the step's runtime due to compiling gperf is smaller relative to the total time it takes for the step to complete.

Reply:
ah, the coverage step uses --all-features and so we run into the same problem of needing to export the path to the next step so that the bundled build can find it.

I'll just have it run in the initial lint step then, unless you think we should add a separate step just to test the install script.

Reply:
... the clippy step also uses --all-features. I'll just add an extra step to test the build script.

Reply:
Sorry for all the build failures. I ended up just adding a single job to test the install script by itself, and then the case where gperf is installed via apt is tested in the other jobs.